site/content/docs/CICD/vercel.md

```diff
@@ -99,10 +140,10 @@ These ranges are listed in the table below and are based on the guidelines devel
 
 | KPI | good       | needs improvement  | poor     |
 |-----|------------|--------------------|----------|
-| FCP | <= 1.8 sec | > 1.8 sec <= 3 sec | > 3 sec  |
-| LCP | <= 2.5 sec | > 2.5 sec <= 4 sec | > 4 sec  |
+| FCP | <= .93 sec | > .93 sec <= 1.6 sec | > 1.6 sec  |
```

Comment:
Can't comment on these so I will assume they are correct.

Reply:
@ragog yeah, this is straight from the source code.